### ISSUE TITLE
Card with video > Anchor: Uncaught TypeError: Cannot read property 'apply' of undefined 

### ISSUE THREAD

**Issue by alampros** [grommet/grommet#1552]
Occurs when clicking the play button on a Card with a video source (even [in the docs](https://grommet.github.io/docs/card/examples?thumbnail=true&label=true&heading=true&description=true&headingStrong=true&video=true&link=true))

### Expected Behavior

The video should play in a `<Layer>`.

### Actual Behavior

An unhandled error is thrown in the console:
```
Anchor.js:258 Uncaught TypeError: Cannot read property 'apply' of undefined
    at onClick (Anchor.js:258)
    at Object../node_modules/react-dom/lib/ReactErrorUtils.js.ReactErrorUtils.invokeGuardedCallback (ReactErrorUtils.js:69)
    at executeDispatch (EventPluginUtils.js:85)
    at Object.executeDispatchesInOrder (EventPluginUtils.js:108)
```

### URL, screen shot, or Codepen exhibiting the issue

 - [Grommet docs](https://grommet.github.io/docs/card/examples?thumbnail=true&label=true&heading=true&description=true&headingStrong=true&video=true&link=true)
 - [Codepen demo](https://codepen.io/alampros/pen/brqxja?editors=0010)

### Steps to Reproduce
 1. Add a `<Card>` component with a `video` prop
 2. Click the play button
 3. Observe the error in the console

### Your Environment

* Grommet version: 1.6.0
* Browser Name and version: Chrome 61.0.3163.31 (also tested in 62.0.3179.0)
* Operating System and version (desktop or mobile): macOS 10.12.6 (desktop)

I also quickly tested this in grommet@1.4.1 and it doesn't throw the error, however, the video does not play.

**Comment by karatechops** [comment]
Hey @alampros I've submitted PR #1591 to fix issue #1585 however it should also fix this issue. Let's see if this still happens after the PR is merged and we can go from there.

**Comment by alansouzati** [comment]
Thanks for fixing this @karatechops. I'm going to close this issue and reopen if the problem persists
